core/tests/test_apps.cardano.sign_tx.py

```diff
@@ -9,7 +9,7 @@
 @unittest.skipUnless(not utils.BITCOIN_ONLY, "altcoin")
 class TestCardanoSignTransaction(unittest.TestCase):
     def test_should_show_outputs(self):
-        outputs_to_show = [
+        outputs_to_hide = [
```

Comment:
those tests apparently were written for a function called `_should_show_output` and were not updated correctly during the refactor to `_should_hide_output`, ending up with a contradictory naming